src/CharterManager.sol

```diff
@@ -117,7 +117,7 @@ contract CharterManagerImp {
 
     // --- Implementation Storage ---
     mapping (address => address) public proxy; // UrnProxy per user
-    mapping (address => mapping (address => uint256)) public can;
+    mapping (address => mapping(address => uint256))  public can;
```

Comment:
I think actually this line was fine as it was before.

Reply:
see line 123

Reply:
actually, this aligns with the others, but yeah, doesn't match the first mapping format. I'll fix the other ones.

Reply:
47c904bf30b5620d497d44cee000cd0eff97d4f2

Reply:
solidity docs are pretty loose on whether it's

`mapping(k => v)`

or 

`mapping (k => v)`

So I guess we just need to pick one and be consistent.